iOS/Flows/Room/RoomCoordinator+Presentation.swift

```diff
@@ -21,7 +21,7 @@ extension RoomCoordinator {
                                                       conversationMembers: [],
                                                       startingConversationId: cid,
                                                       startingMessageId: messageId,
-                                                      openReplies: true)
+                                                      openReplies: openReplies)
```

Comment:
Because Ill be adding a reply to footer, its less useful to deep link into a thread.